src/client/components/ui/Link/Link.test.tsx

```diff
@@ -8,28 +8,22 @@ import { render, screen } from '@testing-library/react';
 import { Link } from '~/client/components/ui/Link/Link';
 
 describe('Link', () => {
-	afterEach(() => {
-		jest.clearAllMocks();
-	});
+	it('quand le lien est un lien externe, retourne un *tag a* avec les propriétés target et rel', () => {
+		const lienExterne = 'https://mon-lien-externe';
 
-	describe('quand le lien est un lien externe', () => {
-		it('retourne le composant Link avec un *tag a* directement et les propriétés target et rel', () => {
-			const lienExterne = 'https://mon-lien-externe';
+		render(
+			<Link href={lienExterne} />,
+		);
 
-			render(
-				<Link href={lienExterne} />,
-			);
-
-			const linkComponent = screen.getByRole('link');
+		const linkComponent = screen.getByRole('link');
 
-			expect(linkComponent.getAttribute('href')).toEqual('https://mon-lien-externe');
-			expect(linkComponent).toHaveAttribute('target');
-			expect(linkComponent).toHaveAttribute('rel');
-		});
+		expect(linkComponent.getAttribute('href')).toEqual('https://mon-lien-externe');
+		expect(linkComponent).toHaveAttribute('target');
+		expect(linkComponent).toHaveAttribute('rel');
 	});
 
-	describe('quand le lien est un lien interne avec href relatif', () => {
-		it('retourne le composant Link sans les propriétés de la redirection externe', () => {
+	describe('lien interne', () => {
```

Comment:
"quand le lien est un lien interne" pour reprendre le même format que le it précédent 
(github est vraiment trop nul ils ont enlevé le bouton suggestion 😭)